client/src/App.tsx

```diff
@@ -6,11 +6,13 @@ import ChannelStatus from './channel/components/ChannelStatus';
 import DatasetList from './dataset/components/DatasetList';
 import ErrorList from './errors/components/ErrorList';
 import logo from './images/LiberTEM logo-medium.png';
+import QuitButton from './quit/components/QuitButton';
 
 class App extends React.Component {
     public render() {
         return (
             <Container style={{ margin: "5em 1em 5em 1em" }}>
+                <QuitButton/>
```

Comment:
Hm, maybe there can be a menu where we put this button? Possibly we should also move the "about"-action from the litte "i"-icon to that menu, to make it more visible/accessible.

Reply:
Yes, so it can be easily accessed even if we scroll to bottom.

Reply:
Yes, also adding "top-level" user interface elements can clutter up the interface - it is better to create a hierarchy of user interface elements

Reply:
Ah, something else: maybe pressing "quit" should also [show a modal dialog](https://react.semantic-ui.com/modules/modal/), where the user can confirm that they want to stop the server? Otherwise, mis-clicks could make people unhappy...